test/Microsoft.AspNetCore.Mvc.FunctionalTests/FiltersTest.cs

```diff
@@ -554,5 +554,53 @@ public FiltersTest(MvcTestFixture<FiltersWebSite.Startup> fixture)
                 $"CurrentCulture:{expected},CurrentUICulture:{expected}",
                 await response.Content.ReadAsStringAsync());
         }
+
+        [Fact]
+        public async Task RequestSizeLimit_ThrowsForInvalidSize()
```

Comment:
If all we do is set a feature, do we need functional tests for this? Wouldn't unit tests suffice?